aws/transit_gateway_types.go

```diff
@@ -0,0 +1,94 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// TransitGatewaysVpcAttachment - represents all transit gateways vpc attachments
+type TransitGatewaysVpcAttachment struct {
+	Ids []string
+}
+
+// ResourceName - the simple name of the aws resource
+func (tgw TransitGatewaysVpcAttachment) ResourceName() string {
+	return "transit-gateway-attachment"
+}
+
+// MaxBatchSize - Tentative batch size to ensure AWS doesn't throttle
+func (tgw TransitGatewaysVpcAttachment) MaxBatchSize() int {
+	return 200
+}
+
+// ResourceIdentifiers - The Ids of the transit gateways
+func (tgw TransitGatewaysVpcAttachment) ResourceIdentifiers() []string {
+	return tgw.Ids
+}
+
+// Nuke - nuke 'em all!!!
```

Comment:
Should this comment be:

```
// Nuke - It's time to kick ass and chew bubblegum, and I'm all out of gum.
```

🤣😂

Duke Nukem reference if you're unfamiliar with that quote.

Reply:
🤣😂🤣😂🤣😂